code/modules/projectiles/guns/energy/laser/disco_vazer.dm

```diff
@@ -1,8 +1,6 @@
 /obj/item/gun/energy/lasersmg
-	name = "Disco Vazer \"Lasblender\""
-	desc = "This conversion of the \"Texan\" that enables it to shoot lasers. Unlike in other laser weapons, the process of creating a laser is based on a chain reaction of localized micro-explosions. \
```

Comment:
soulless removal. disco vaser description and name ('lasblender' is AWESOME) is cool.

Reply:
I don't care about soul. Most of us don't. Things change.